src/calculator.js

```diff
@@ -3,6 +3,14 @@
 var Precedence = require('./precedence');
 var Screen = require('./screen');
 var Memory = require('./memory');
+var EventDispatcher = require('./eventDispatcher');
```

Comment:
Why have you added this file to the pull request? This is just some debug / testing code which must not be a part of repositorium, revert the change.